front/lib/resources/vault_resource.ts

```diff
@@ -156,11 +156,14 @@ export class VaultResource extends BaseResource<VaultModel> {
   }
 
   static async listWorkspaceVaults(
-    auth: Authenticator
+    auth: Authenticator,
+    adminsBypassACL: boolean = true
```

Comment:
Let's not do this even more with a default value set to `true`. See https://github.com/dust-tt/dust/issues/7146.